infrastructure/smart-contracts/test/Fixture.mjs

```diff
@@ -322,14 +322,31 @@ describe("Fixture Tests", function () {
         const poolFactory2 = await upgrades.upgradeProxy((await poolFactory.getAddress()), PoolFactory2, { call: { fn: "initialize", args: [await tinyKeysAirDrop.getAddress()] } });
         await poolFactory2.waitForDeployment();
 
+        //Deploy sample USDC token
+        const usdcName = "Test USDC";
+        const usdcSymbol = "USDC";
+        const usdcToken = await ethers.deployContract(
+            "SampleERC20",
+            [usdcName, usdcSymbol]
+        );
+        await usdcToken.waitForDeployment();
         // // Node License8 Upgrade - Required For Tiny Keys
         const NodeLicense8 = await ethers.getContractFactory("NodeLicense8");
-        const nodeLicense8 = await upgrades.upgradeProxy((await nodeLicense.getAddress()), NodeLicense8, { call: { fn: "initialize", args: [await xai.getAddress(), await esXai.getAddress(), await chainlinkEthUsdPriceFeed.getAddress(), await chainlinkXaiUsdPriceFeed.getAddress(), await tinyKeysAirDrop.getAddress()] } });
+        const nodeLicense8 = await upgrades.upgradeProxy((await nodeLicense.getAddress()), NodeLicense8, { call: { fn: "initialize", args: [await xai.getAddress(), await esXai.getAddress(), await chainlinkEthUsdPriceFeed.getAddress(), await chainlinkXaiUsdPriceFeed.getAddress(), await tinyKeysAirDrop.getAddress(), await usdcToken.getAddress()] } });
         await nodeLicense8.waitForDeployment();
 
         // Setup admin mint to role for nodeLicenseDefaultAdmin
         await nodeLicense8.connect(nodeLicenseDefaultAdmin).grantRole(await nodeLicense8.ADMIN_MINT_ROLE(), nodeLicenseDefaultAdmin.address);
 
+        // // Node License8 Upgrade - Required For Tiny Keys
+        // const NodeLicense8 = await ethers.getContractFactory("NodeLicense8");
+        // const nodeLicense8 = await upgrades.upgradeProxy(
+        //     (await nodeLicense.getAddress()), 
+        //     NodeLicense8, 
+        //     { call: { fn: "initialize", args: [await xai.getAddress(), await esXai.getAddress(), await chainlinkEthUsdPriceFeed.getAddress(), await chainlinkXaiUsdPriceFeed.getAddress(), await tinyKeysAirDrop.getAddress(), await usdcToken.getAddress()] } });
+        // await nodeLicense8.waitForDeployment();
```

Comment:
I don't think this comment is needed in here anymore

Reply:
Removed